pkg/kubeapiserver/options/authentication.go

```diff
@@ -260,6 +261,9 @@ func (s *BuiltInAuthenticationOptions) AddFlags(fs *pflag.FlagSet) {
 		fs.StringSliceVar(&s.ServiceAccounts.APIAudiences, "service-account-api-audiences", s.ServiceAccounts.APIAudiences, ""+
 			"Identifiers of the API. The service account token authenticator will validate that "+
 			"tokens used against the API are bound to at least one of these audiences.")
+
+		fs.DurationVar(&s.ServiceAccounts.MaxExpTime, "max-service-account-token-expiration", s.ServiceAccounts.MaxExpTime,
```

Comment:
nit: s/MaxExpTime/MaxExpiration/

it isn't a time and we shouldn't abbreviate to Exp here.